flfm/src/java/pytorch/flfm/src/test/java/ai/jhu/edu/flfm/AppTest.java

```diff
@@ -0,0 +1,19 @@
+package ai.jhu.edu.flfm;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest {
+
+    /**
+     * Rigorous Test :-)
+     */
+    @Test
+    public void shouldAnswerWithTrue() {
+        assertTrue(true);
```

Comment:
🤨

Reply:
Is it not trivial to do something akin to the following (with appropriate args)?

```java
CommandLine.run(new App(), args);
```

Reply:
That test is the default test.

You could run it, but you would need to export the model first. Which would make it depend on the exporting code. Do you want to save an exported version of the model to run this?